code/game/turfs/open.dm

```diff
@@ -119,8 +142,10 @@
 					step(C, olddir)
 					C.spin(1,1)
 		else if(lube&SLIDE_ICE)
+			C.slipping = TRUE
 			spawn(1)
 				step(C, olddir)
```

Comment:
it is your duty to refactor slipping code to support this effect then, not replicating the functionality in a different location.

I will not warn you to adjust your attitude a final time.